packages/composite-checkout-wpcom/src/lib/edit-cart.js

```diff
@@ -0,0 +1,8 @@
+export function replacePlanWithDifferentLength() {}
+
+export function adjustItemPricesForCountry() {}
+
+// replace this once https://github.com/Automattic/wp-calypso/pull/36758 is merged
+export function formatValueForCurrency( currency, value ) {
```

Comment:
Let's add a safeguard here so we don't forget about this by adding `if ( currency !== 'USD' ) { throw new Error( 'Non-USD currency is not yet supported' ); }`